app/assets/javascripts/battletype/ship.js

```diff
@@ -11,13 +11,35 @@
         }
       },
       velocity: {
-        get: function () { return this._word; },
+        get: function () { return this._velocity; },
         set: function (v) { this._velocity = v; }
       },
       damage: {
         get: function () { return this._damage; },
         set: function (d) { this._damage = d; }
       },
+      height: {
+        get: function () {
+          if (this._height) {
+            return this._height;
+          } else {
+            // FIXME: Find a better way to get this value.
```

Comment:
We could try the [`Element.getBoundingClientRect()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) method. But do we really need a public _getter_ for this property? And a _setter_?